spec/text_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'text'
+require 'twilio-ruby'
+
+describe Text do
+
+  it 'can send message to confirm order' do
+    messages = double :messages, create: nil
+    account = double :account, messages: messages
+    client = double :client, account: account
+
+    allow(Twilio::REST::Client).to receive(:new) { client }
+    expect(messages).to receive(:create).with(
+    body: "Your order has been confirmed. It will be delivered before" \
+      " #{(Time.new + 3600).strftime("%H:%M")}",
+    to: 447400606699,
+    from: 441793250274)
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).